### PR TITLE
Removing star exports from @fluentui/react-avatar

### PR DESCRIPTION
**Pull request by khmakoto**
## Current Behavior

`react-avatar` has `export * from ...` in `src/index.ts`.

## New Behavior

`react-avatar` has explicitly named exports in `src/index.ts`.

## Related Issue(s)

#22099

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    | `44.982 kB`<br />`13.059 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `175.69 kB`<br />`48.974 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            | `32.526 kB`<br />`10.645 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/80746a7f3462c2fa4ff15a92d6e196c67d652f6a'>80746a7f3462c2fa4ff15a92d6e196c67d652f6a</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-mount_base.html">297</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-mount.html">295</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/Avatar-mount_base.html">890</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/Avatar-mount.html">914</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/Button-mount_base.html">554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/Button-mount.html">545</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProvider-mount_base.html">1849</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProvider-mount.html">1839</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-mount_base.html">297</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-mount.html">295</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">222</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">217</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">283</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">317</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/MakeStyles-mount_base.html">1565</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22113/perf-test-react-components/MakeStyles-mount.html">1568</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by ecraig12345 — APPROVED** [review]

File: packages/react-avatar/src/index.ts [hunk 1]
```diff
@@ -1,2 +1,3 @@
-export * from './Avatar';
-export * from './utils/index';
+export { Avatar, avatarClassName, renderAvatar_unstable, useAvatarStyles_unstable, useAvatar_unstable } from './Avatar';
```
Comment (spmonahan) [inline packages/react-avatar/src/index.ts:1]: Avatar now exports [avatarClassNames](https://github.com/microsoft/fluentui/blob/master/packages/react-avatar/src/components/Avatar/useAvatarStyles.ts#L10)

Comment (khmakoto) [reply packages/react-avatar/src/index.ts]: Added.

**Pull request merged**